frameworks/Java/ratpack/ratpack.dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM gradle:4.7.0-jdk8 as gradle
```

Comment:
Presumably, this puts `gradlew` on the path, and therefore the inclusion of that file in this codebase is unnecessary.

Reply:
See above